libs/langchain/langchain/embeddings/cohere.py

```diff
@@ -87,7 +98,7 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
             List of embeddings, one for each text.
         """
         embeddings = self.client.embed(
-            model=self.model, texts=texts, truncate=self.truncate
+            model=self.model, texts=texts, truncate=self.truncate, input_type=self.input_type
```

Comment:
will it break things for v2 models to pass input_type?

Reply:
it will just be ignored, have tested